build_projects/update-dependencies/Config.cs

```diff
@@ -35,6 +36,7 @@ public class Config
         private Lazy<string> _password = new Lazy<string>(() => GetEnvironmentVariable("GITHUB_PASSWORD"));
 
         private Lazy<string> _coreFxVersionUrl = new Lazy<string>(() => GetEnvironmentVariable("COREFX_VERSION_URL", "https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/release/1.0.0/Latest_Packages.txt"));
+        private Lazy<string> _coreClrVersionUrl = new Lazy<string>(() => GetEnvironmentVariable("CORECLR_VERSION_URL", "https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/coreclr/release/1.0.0/Latest_Packages.txt"));
```

Comment:
I was able to remove it:  https://github.com/dotnet/core-setup/pull/83/commits/94b1ad59d02d85802f41d76a6f6f0e3f4a9e45e0

The build succeeded on my machine without this dependency.  So core-setup is not dependent on any projectk-tfs packages.